index.html

```diff
@@ -1,19 +1,88 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
     <title>Karma</title>
-    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
-<body>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+
+  </head>
+  <body>
+    <!--header-->
+    <header>
+      <a href="#"><img  class="karma__logo" src="img/karma-logo.svg" alt="Karma Logo" /></a>
+
+    <!-- Navigation Bar -->
+      <nav>
+        <ul id="navigation__list">
+          <li class="nav__item">
+            <a href="#" class="nav__item-link">Meet Karma</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__item-link">How it Works</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__item-link">Store</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__item-link">Blog</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__item-link">Help</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__item-link">Login</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+
+   <!-- Hero Section -->
+      <section id="hero">
+        <h1>Introducing Karma</h1>
+        <h2>Bring WiFi with you, everywhere you go.</h2>
+        <button>Learn More</button>
+      </section>
+
+    <!--Icons Section -->
+      <section id="icons">
+        <h1>Everyone needs a little Karma.</h1>
+        <div class="icon__images">
+          <img src="img/icon-devices.svg" alt="device image" />
+          <img src="img/icon-coffee.svg" alt="coffee cup image" />
+          <img src="img/icon-refill.svg" alt=" ipod image" />
+          <h4>Internet for all devices</h4>
+          <h4>Boost your productivity</h4>
+          <h4>Pay as you go</h4>
+        </div>
+      </section>
+    </main>
+
```

Comment:
You should have an `<hr/>` tag here with the right color to divide the content as shown in the sample image